conjure-java-core/src/integrationInput/java/com/palantir/product/EteServiceRetrofit.java

```diff
@@ -112,4 +113,28 @@
     @POST("./base/no-return")
     @Headers({"hr-path-template: /base/no-return", "Accept: application/json"})
     Call<Void> noReturn(@Header("Authorization") AuthHeader authHeader);
+
+    @GET("./base/enum/query")
+    @Headers({"hr-path-template: /base/enum/query", "Accept: application/json"})
+    Call<SimpleEnum> enumQuery(
+            @Header("Authorization") AuthHeader authHeader,
+            @Query("queryParamName") SimpleEnum queryParamName);
+
+    @GET("./base/enum/list/query")
+    @Headers({"hr-path-template: /base/enum/list/query", "Accept: application/json"})
+    Call<List<SimpleEnum>> enumListQuery(
+            @Header("Authorization") AuthHeader authHeader,
+            @Query("queryParamName") List<SimpleEnum> queryParamName);
```

Comment:
I don't think we were intending to support Lists or Sets in query / header paramters?

> If the param-type is path, header or query then the de-aliased type MUST be an enum, a primitive or an optional of one of these. Additionally, the type MUST NOT be bearertoken when the param-type is header or query

https://palantir.github.io/conjure/#/docs/spec/conjure_definitions

Reply:
We already support both lists and sets in query parameters, we use them heavily across the board, this provides parity with the jersey implementation.

Happy to update the spec, but I'd prefer not to block this change on it.